CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
+## [unreleased]
```

Comment:
Please run git cliff with `--bump` to add the proper version here

Reply:
I edited manually because with bump the minor was pushed to 0.8.0; weird, I thought it would update the patch without bump.

Reply:
Is that ok?

Reply:
I think we want to set those options to `False` to handle `0.x.x` releases more gracefully: https://git-cliff.org/docs/configuration/bump

Could you try that and see if it fixes the issue?

Reply:
That fixed it 👍 edited cliff.toml